Makefile

```diff
@@ -189,13 +189,13 @@ version: login-gh update-kustomize-version ## Update dependent versions
 	$(call update-version,kubernetes-sigs/kind,KIND_VERSION)
 	$(call update-version,rust-lang/mdBook,MDBOOK_VERSION)
 
-	$(call update-version-quay,cert-manager,CERT_MANAGER_VERSION)
-	$(call update-version-quay,cilium,CILIUM_VERSION)
+	$(call update-version-ghcr,cert-manager,CERT_MANAGER_VERSION)
+	$(call update-version-ghcr,cilium,CILIUM_VERSION)
```

Comment:
`cilium` container image doesn't exist in ghcr.io now.
So we have to wait for creating it or keep using quay.io until migrating cilium's image.

Reply:
@terassyi 
Thanks for the reminder. Does it ok to proceed now?

Reply:
Sorry for my late reply. 
It's ok to do.